gapir/cc/function_table.h

```diff
@@ -49,20 +54,26 @@ class FunctionTable {
   inline Function* lookup(Id id);
 
  private:
-  // Map of the supported function ids to the actual function implementations
-  std::unordered_map<Id, Function> mFunctions;
+  // Array of the actual function implementations by function ID
+  // This is stored as an array rather than a map because many lookups are
+  // performed at replay time and this becomes a bottleneck when stored as a
+  // map. The 64k entries (limit mandated elsewhere in the code by the vm
+  // bytecode instruction packing) are small enough that storing them as an
+  // array isn't a problem. These are dynamically allocated because
+  // windows fails at compile time with an inline array.
+  Function* mFunctions;
```

Comment:
This should either have been a std::vector or std::unique_ptr<>. It should not have been a raw pointer if we can help it.